sections/shared/Layout/AppLayout/Header/UserMenu/getUserENSNameAndAvatarUrl.ts

```diff
@@ -0,0 +1,10 @@
+export default async function getENSNameAndAvatarUrl(_account: string, _provider: any) {
```

Comment:
Can we rename this to `getENSName` because it doesn't look like we're doing anything with the avatar